Sources/Apollo/GraphQLOperation.swift

```diff
@@ -1,6 +1,7 @@
 public protocol GraphQLOperation: class {
   static var operationString: String { get }
   static var requestString: String { get }
+  static var operationIdentifier: String { get }
```

Comment:
The problem is that the generated `operationIdentifier`s would then have to be `String?`s as well, which doesn't make much sense from the standpoint of the generated code.